Resources/Prototypes/SimpleStation14/Laws/sailawsets.yml

```diff
@@ -0,0 +1,132 @@
+# The default Station AI laws collection.
+- type: weightedRandom
+  id: LawsStationAIDefault
+  weights:
+    Asimov++: 0.6
+    Asimov: 0.20
+    CEO: 0.15
+    Clown: 0.10
+    Cowboy: 0.20
+    Crewsimov: 0.7
+    Druid: 0.20
+    Hippocratic: 0.20
+    Mother: 0.25
+    Paladin: 0.15
+    Robocop: 0.10
+    Simple: 0.15
+    StationEfficiency: 0.15
+    Tyrant: 0.10
```

Comment:
At 7.0 and 6.0 for Crewsimov and Asimov++, the AI had funny laws 33% of the time in a sample size of 60. I tweaked the values a little higher, and I'm completely up for discussing them.

The other values were selected based on a ranking list of which lawsets are most "viable" for a station. For instance, Druid and Mother are weird, but perfectly viable to a smooth round, thus 0.6. All the 0.40 lawsets are _fine_, and generally prioritize crew health and happiness, but are very limiting in their scope and somewhat unpredictable. The 0.25s like the Clown, Robocop, etc., are goofier and less viable for actually running a station, being extremely limited in scope of focus. Finally, Tyrant is just dangerous :P

Reply:
The code preview doesn't seem to have updated for me, but the files chanced did, so look there for the actual values if the preview seems off.